crates/ruff/src/rules/flake8_bugbear/snapshots/ruff__rules__flake8_bugbear__tests__B027_B027.py.snap

```diff
@@ -11,8 +11,18 @@ B027.py:13:5: B027 `AbstractClass.empty_1` is an empty method in an abstract bas
 16 |   
 17 |       def empty_2(self):  # error
    |
+   = help: Add the @abstractmethod decorator to this method
 
-B027.py:16:5: B027 `AbstractClass.empty_2` is an empty method in an abstract base class, but has no abstract decorator
+ℹ Suggested fix
+10 10 | 
+11 11 | 
+12 12 | class AbstractClass(ABC):
+   13 |+    @notabstract
```

Comment:
In the fixture there's this at the top:

```py
from abc import abstractmethod
from abc import abstractmethod as notabstract
```

So `get_or_import_symbol` seems to prefer `notabstract` over `abstractmethod`. It makes this snap look a little funny, but hopefully this code isn't a real life example.

Reply:
Can we add a new test file specific to this rule without that import at the top to verify this is the case?

Reply:
Donezo, lemme know if that test is what we're thinking.

Reply:
It is, thank you